### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kushal34712**
fixed typo and corrected grammer

### CODE REVIEW

**Comment by qgallouedec** [comment]
I let native English speakers review this one 😉

**Comment by kushal34712** [comment]
@qgallouedec why not sir,  just check it once and merge it if possible

**Comment by kushal34712** [comment]
@qgallouedec sir it its reviewed then merge it please

**Comment by qgallouedec** [comment]
Hi @kushal34712 as I've written, I've requested the review of native english speakers. It will be merged when reviewed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,19 +74,19 @@ If there is a new feature you'd like to see in TRL, please open an issue and des
...
-3. Provide a *code snippet* that demonstrates the features usage.
+3. Provide a *code snippet* that demonstrates the feature's usage.
 4. If the feature is related to a paper, please include a link.
 
 If your issue is well written we're already 80% of the way there by the time you create it.
 
 ## Do you want to implement a new trainer?
 
-New post-training methods are published on a frequent basis and those which satisfy the following criteria are good candidates to be integrated in TRL:
+New post-training methods are published frequently and those that satisfy the following criteria are good candidates to be integrated in TRL:
```
Comment (kashif) [inline CONTRIBUTING.md:84]:
```suggestion
New post-training methods are published frequently and those that satisfy the following criteria are good candidates to be integrated into TRL:
```

**Review by kashif — APPROVED** [review]

**Review by edbeeching — APPROVED** [review]
LGTM, thanks @kushal34712 for your patience. We aim to review PRs in detail (even doc changes) so please allow us the time to do this.

**Review by lewtun — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**